user/utils.py

```diff
@@ -1,11 +1,32 @@
 import json
+import jwt
+import bcrypt
 
 from django.http import JsonResponse
 
+from my_settings import SECRET_KEY, ALGORITHM
+from user.models import User
+
 def ValueErrorTypeChecking(data):
     if type(data['email']) != str:
         return JsonResponse({'MESSAGE' : 'INVALID_EMAIL_TYPE'}, status=400)
 
     if type(data['password']) != str:
         return JsonResponse({'MESSAGE' : 'INVALID_PASSWORD_TYPE'}, status=400)
 
+    if type(data['phone_number']) != str:
+        return JsonResponse({'MESSAGE' : 'INVALID_PHONE_NUMBER_TYPE'}, status=400)
```

Comment:
`isinstance()` 사용하여, True, False 값을 반환 받아서 사용할 수 있습니다.

```suggestion
    if not isinstance(data['email'], str):
        return JsonResponse({'MESSAGE' : 'INVALID_EMAIL_TYPE'}, status=400)
    if not isinstance(data['password'], str):
        return JsonResponse({'MESSAGE' : 'INVALID_PASSWORD_TYPE'}, status=400)
```

Reply:
반영했습니다. 감사합니다!